### ISSUE TITLE
Add Corp metrics option

### ISSUE THREAD

**Issue by resixh** [dysath/seat-srp#79]
Please make it a option to see how much srp goes by for corps next to the users in reports

**Comment by warlof** [comment]
Meaning at alliance point of view, getting metrics in a dashboard which tell you per corporation how much SRP has been granted ?

**Comment by resixh** [comment]
Yes so the corps can see how much they use or get out of their tax money in value
